app/controllers/api/v1/keywords_controller.rb

```diff
@@ -7,7 +7,7 @@ class KeywordsController < ApplicationController
       include API::V1::Pagy::JSONAPIConcern
 
       def index
-        pagy, keywords_list = pagy(keywords)
+        pagy, keywords_list = pagy(keywords_query[:keywords])
```

Comment:
I think these changes should be in the API task

Reply:
@junan It's not adding a new thing, but handling non regression: If I put this change in the API branch, it means we cannot release the backend branch independently. I think it is not a safe idea to extends a non-working part across PR (unless really no choice). 
Note that url_match_count is not added in here, as this indeed is a new feature and so it shall be done in a separate branch. 
Not sure if I have convinced you @junan, let me know :)

Reply:
For this case, I am fine as you changed the core class(KeywordsQuery), without updating here, the existing API code will not work. 

Usually, we try to do break things(tasks) separate PRs whenever possible. And regarding releasing each PR independently, I think we don't need it. Just like we will not release only the backend PR changes without front end PR changes and for releasing a big single feature, we will need to work on multiple PRs and then will release when all after all PRs merged of that big single feature.